packages/combobox/src/Combobox/Combobox.styles.ts

```diff
@@ -261,7 +270,7 @@ export const inputWrapperStyle = ({
         gap: ${flexGap}px;
         overflow-x: hidden;
         min-height: ${inputHeight(size)}px;
-        max-height: calc((${getChipHeight(size) * 3}px) + (${flexGap}px * 2));
+        max-height: calc((${inputHeight(size) * 3}px) + (${flexGap}px * 2));
```

Comment:
Not from your PR, but it'd be nice to know what the `3` is for

Reply:
😅